src/runtime/manifest-parser.peg

```diff
@@ -352,7 +352,7 @@ MetaStorageKey = 'storageKey' whiteSpace? ':' whiteSpace? key:id eolWhiteSpace
 };
 
 Particle
-  = 'particle' whiteSpace name:upperIdent verbs:(whiteSpace VerbList)? implFile:(whiteSpace 'in' whiteSpace id)? eolWhiteSpace items:(Indent (SameIndent ParticleItem)*)? eolWhiteSpace?
+  = external:('external' whiteSpace)? 'particle' whiteSpace name:upperIdent verbs:(whiteSpace VerbList)? implFile:(whiteSpace 'in' whiteSpace id)? eolWhiteSpace items:(Indent (SameIndent ParticleItem)*)? eolWhiteSpace?
```

Comment:
Please update the 'ReservedWords' list. This will ensure that the following yields the right kind of error (we can relax this when the unified syntax is default, as the colons will protect against misunderstandings).
```
particle Foo
  external particle Bar
```

Reply:
Done